input/pagecontent/concepts_onboarding_checklist.md

```diff
@@ -117,7 +117,8 @@ This repository contains the template for building [onboarding](https://github.c
 
 - You must adapt the following default certificate parameter of [DN_template.cnf](https://github.com/WorldHealthOrganization/tng-participant-template/blob/main/scripts/certgen/DN_template.cnf) file which will used  in gen_all_certs.sh to your needs:
   
-- Configuration Template for Certificate Generation, Modify for your own needs
+- Configuration Template for Certificate Generation, Modify for your own needs in DN_template.cnf file as it will 
+  used argumnet while running the script "gen_all_certs.sh".
```

Comment:
as it will be used as argument

Please add grammar @arajnor99

Reply:
correction done
https://github.com/WorldHealthOrganization/smart-trust/blob/42328c7cdef638362ab83bb2e2952f23bd918e66/input/pagecontent/concepts_onboarding_checklist.md?plain=1#L120